src/robusta/core/sinks/robusta/dal/model_conversion.py

```diff
@@ -146,6 +147,8 @@ def to_evidence_json(
                     )
 
                 structured_data.append({"type": "callbacks", "data": callbacks})
+            elif isinstance(block, JsonBlock):
```

Comment:
how does this look in the UI and slack?

Reply:
Also this might send JsonBlock to all the sinks now, there might have been a reason why this wasnt allowed until now, maybe check where the finding is sent without this part?

Reply:
yes, because it why would you send json to slack